src/api/status/public/css/terminal.css

```diff
@@ -1,3 +1,6 @@
 .xterm {
   padding: 0.5rem;
+  position: relative;
+  width: 100vh;
```

Comment:
These two doesn't seem to be necessary. `vh` should not be used for width as well.